### ISSUE TITLE
Do we really need to export vs and core from VapourSynth?

### ISSUE THREAD

**Issue by stuxcrystal** [Irrational-Encoding-Wizardry/vsutil#31]
From the original commit, <https://github.com/Irrational-Encoding-Wizardry/vsutil/pull/28/commits/45d22b76caa03bd3b785fe22a0cbd611ceb79246>
> This allows cleaner looking scripts, and might as well be the
recommended way of import from vsutil _in scripts_ not other modules.
With `core` and `vs` being defined in __all__, a VS script will not need
both of the standard VS lines if using the wild card import.

In someone's encoding script, a `from vsutil import *` will allow them to access all `vs` members and use the normal `core.Etc.etc` syntax without having to write:

```py
import vapoursynth as vs
from vsutil import *
core = vs.core
```
at the top of every script.

_Originally posted by @OrangeChannel in https://github.com/Irrational-Encoding-Wizardry/vsutil/pull/29_

**Comment by kageru** [comment]
Since you’re asking that question in an issue, I assume you don’t agree with @OrangeChannel.
Would you mind explaining your reasoning behind that?

**Comment by stuxcrystal** [comment]
I personally thing, script-writers and library-writers will continue using the code (or a variant of it):

```py
import vapoursynth as vs
from vapoursynth import core
```

The decision to import `vsutil` will probably come later and thus is unlikely to change how `vs` and `core` are being imported.

As such, I think this feature is unlikely to be used by any of the intended audiences (script-writers and library-writers) and thus can be safely removed.

**Comment by OrangeChannel** [comment]
I'm hesistant to agree but I see your point in
> The decision ... will probably come later

However, I would like address "As such, I think this feature is unlikely to be used by any of the intended audiences (script-writers and library-writers) and thus can be safely removed."

Library writers shouldn't be wildcard importing from *any* modules really, this is dangerous and makes reading the code to an outsider even more confusing.

**Comment by kageru** [comment]
Closed by #34
